core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -597,7 +610,7 @@ protected RpcOptions createRpcOptions() {
 
    private class TxReadOnlyManyHelper extends ReadOnlyManyHelper {
       @Override
-      public ReplicableCommand copyForRemote(ReadOnlyManyCommand command, List<Object> keys, InvocationContext ctx) {
+      public ReadOnlyManyCommand<?, ?, ?> copyForRemote(ReadOnlyManyCommand command, List<Object> keys, InvocationContext ctx) {
```

Comment:
Java gives us the liberty to use or not use generics when appropriate. In the stack we have no clues about key/value types, so it's better to omit them here.